### PR TITLE
040 login system

### PR DESCRIPTION
**Pull request by eoinmcbennett**

### CODE REVIEW

**Review by 40175096 — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+  pull_request:
```
Comment (shaunganley) [inline .github/workflows/test.yml]: these still don't specify the main branch, have a look at other examples

File: test/unit/service/authServiceTest.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+const dotenv = require('dotenv');
+
+import login from '../../../service/authService';
+import { expect } from 'chai';
+
+// const AuthService = require("../../../service/authService")
```
Comment (shaunganley) [inline test/unit/service/authServiceTest.ts]: remove if not needed

File: views/index.html [hunk 3]
```diff
@@ -2,20 +2,171 @@
...
     <meta charset="UTF-8">
+    <link rel="icon" type="image/x-icon" href="images/favicon.png">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
     <title>Index</title>
 </head>
+
 <body>
-    <header class="bg-dark text-light">
+    <header id="header" class="header">
```
Comment (shaunganley) [inline views/index.html]: is any of this html re-usable and can be moved to a template that will be imported when needed?

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: views/index.html (hunk 3 above)
Comment (eoinmcbennett) [reply views/index.html]: Yeah just needed to use our new templates from main, sorted now.

File: .github/workflows/test.yml (hunk 1 above)
Comment (eoinmcbennett) [reply .github/workflows/test.yml]: Should be sorted now

**Review by shaunganley — APPROVED** [review]

**Pull request merged**